### ISSUE TITLE
producer-api

### ISSUE THREAD

**Issue by h1alexbel** [eo-cqrs/eo-kafka#10]

**Comment by h1alexbel** [comment]
producer API is ready to go.
Actual implementation:

```java
final Producer<String, String> producer =
      new KfProducer<>(
        new KfProducerSettings<String, String>(
          new XMLDocument(
            new File("${xml file}")
          )
        ).producer()
      );
    producer.send(
      "key-0",
      new KfData<>(
        "test-0",
        "testing",
        1
      )
    );
```

#39 will be implemented after constructing consumer api: #25 , #41

**Comment by h1alexbel** [comment]
also, maybe we need to put some examples of API usage in [README](https://github.com/eo-cqrs/eo-kafka/blob/master/README.md) #36

**Comment by 0pdd** [comment]
@h1alexbel the puzzle [#44](https://github.com/eo-cqrs/eo-kafka/issues/44) is still not solved.

**Comment by 0pdd** [comment]
@h1alexbel 2 puzzles [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293) are still not solved; solved: [#44](https://github.com/eo-cqrs/eo-kafka/issues/44).

**Comment by 0pdd** [comment]
@h1alexbel 3 puzzles [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293), [#364](https://github.com/eo-cqrs/eo-kafka/issues/364) are still not solved; solved: [#44](https://github.com/eo-cqrs/eo-kafka/issues/44).

**Comment by 0pdd** [comment]
@h1alexbel 2 puzzles [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293) are still not solved; solved: [#364](https://github.com/eo-cqrs/eo-kafka/issues/364), [#44](https://github.com/eo-cqrs/eo-kafka/issues/44).

**Comment by 0pdd** [comment]
@h1alexbel the puzzle [#373](https://github.com/eo-cqrs/eo-kafka/issues/373) is still not solved; solved: [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293), [#364](https://github.com/eo-cqrs/eo-kafka/issues/364), [#44](https://github.com/eo-cqrs/eo-kafka/issues/44).

**Comment by 0pdd** [comment]
@h1alexbel all 5 puzzles are solved here: [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293), [#364](https://github.com/eo-cqrs/eo-kafka/issues/364), [#373](https://github.com/eo-cqrs/eo-kafka/issues/373), [#44](https://github.com/eo-cqrs/eo-kafka/issues/44).
